src/mono/wasm/runtime/method-binding.ts

```diff
@@ -43,11 +42,7 @@ export function _get_class_name(classPtr: MonoClass): string {
 }
 
 export function find_method(klass: MonoClass, name: string, n: number): MonoMethod {
-    const result = cwraps.mono_wasm_assembly_find_method(klass, name, n);
```

Comment:
Why?

Reply:
because otherwise it would capture the lazy function instead of resolved function.

Reply:
`_method_descriptions` was not used anywhere

Reply:
OK, it used to get used for diagnostics purposes, if it's not needed anymore then that's good, we must have refactored it out